interview-questions.md

```diff
@@ -8,48 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: 
+    JSX: This is part of the React User Interface web application created by FaceBook. With JSX, developers can create dynamic UI web applications. JSX is a hybrid syntax that merges HTML and JavaScript. For instance, unlike HTML in JSX, a developer can invoke a JS function in the "body" in HTML. 
 
 Researched answer:
+    JavaScript XML, commonly known as JSX, is a JavaScript library that developers use to create code that resembles HTML. Developers use JSX using React to develop complex UI web applications. Although it may look like a developer is running HTML code inside JavaScript, the compilation of code embedded inside the React framework is JavaScript code.
```

Comment:
One of the syntax differences you could mention here would be the use of className instead of class because its a reserved word. Any other syntax differences?